liveHomeFrontend/src/styles/pages/admin/filter.module.sass

```diff
@@ -0,0 +1,12 @@
+@import '../../vars'
+
+.__container 
+  width: 90%
+  margin: 0 auto
+  position: relative
+  
+.__date_select
+  display: grid
+  grid-template-columns: 1fr 1fr
+  grid-gap: 5%
```

Comment:
i am not sure, but i think the gap property in % is incompatible with different browsers